engine/inputdata/blueprint.ftl

```diff
@@ -21,6 +21,238 @@
+                    "Children" : combineEntities(attrs, components, ADD_COMBINE_BEHAVIOUR)
+                }
+            ]]
+        [/#if]
+
+    [/#list]
+
+    [#-- Reference Data --]
+    [#local referenceChildren = []]
+    [#list referenceConfiguration as type,referenceConfig]
+
+        [#if referenceConfig.Type.Plural == "CertificateBehaviours"]
+            [#local referenceChildren += [
+                {
+                    "Names" : referenceConfig.Type.Plural,
+                    "SubObjects" : false,
+                    "Children" : referenceConfig.Attributes
+                }
+            ]]
+        [#else]
+            [#local referenceChildren += [
+                {
+                    "Names" : referenceConfig.Type.Plural,
+                    "SubObjects" : true,
+                    "Children" : referenceConfig.Attributes
+                }
+            ]]
+        [/#if]
+    [/#list]
+
+    [#local settingsChildren = [
+        {
+            "Names" : "Settings",
+            "Children" : [
+                {
+                    "Names" : "Accounts",
+                    "Children" : [
+                        {
+                            "Names" : "*",
+                            "Types" : ANY_TYPE
+                        }
+                    ]
+                },
+                {
+                    "Names" : "Products",
+                    "Children" : [
+                        {
+                            "Names" : "*",
+                            "Types" : ANY_TYPE
+                        }
+                    ]
+                }
+            ]
+        },
+        {
+            "Names" : "Builds",
+            "Children" : [
+                {
+                    "Names" : "Accounts",
+                    "Children" : [
+                        {
+                            "Names" : "*",
+                            "Types" : ANY_TYPE
+                        }
+                    ]
+                },
+                {
+                    "Names" : "Products",
+                    "Children" : [
+                        {
+                            "Names" : "*",
+                            "Types" : ANY_TYPE
+                        }
+                    ]
+                }
+            ]
+        },
+        {
+            "Names" : "Sensitive",
+            "Children" : [
+                {
+                    "Names" : "Accounts",
+                    "Children" : [
+                        {
+                            "Names" : "*",
+                            "Types" : ANY_TYPE
+                        }
+                    ]
+                },
+                {
+                    "Names" : "Products",
+                    "Children" : [
+                        {
+                            "Names" : "*",
+                            "Types" : ANY_TYPE
+                        }
+                    ]
+                }
+            ]
+        }
+    ]]
```

Comment:
I don't know of a way to include Settings in the blueprint. Not sure this one should be here

Reply:
I'm really not sure how / why you would validate them either... 

I've removed them now.